python/oneflow/test/graph/test_graph_io_check.py

```diff
@@ -50,17 +50,17 @@ def fn(*args, **kwargs):
             inp = (args, kwargs)
             print("origin: ", inp)
 
-            io_node = IONode(None, 0, inp, "Graph_0")
+            io_node, named_nodes = construct_io_node(inp, "Graph_0", None)
```

Comment:
Any benefit to change OOP to func?

Reply:
OOP with method is better for understanding and later change. 

This change is better to split from this PR, create a new branch to flowing this refactor.